app/init.php

```diff
@@ -1022,6 +1025,11 @@ function getDevice($root): Device
     return $register->get('geodb');
 }, ['register']);
 
+App::setResource('passwordsDB', function ($register) {
```

Comment:
```suggestion
App::setResource('passwordsDictionary', function ($register) {
```

Let's introduce less new terms

Reply:
Same in other places.